subjects/java/checkpoints/system-logger/README.md

```diff
@@ -0,0 +1,36 @@
+## SystemLogger
+
+### Instructions
+
+In a file named `SystemLog.java` write a function `systemLog` that takes a String message as a parameter and returns the given message concatenated with 'System Log: '
+
+### Expected Functions
+
+```java
+public class SystemLog {
+    public static String systemLog(String message) {
+        // your code here
+    }
+}
+```
+
+### Usage
+
+Here is a possible ExerciseRunner.java to test your function :
```

Comment:
I would keep the same formatting style for the file name `ExerciseRunner.java`

Reply:
done